internal/boxcli/global.go

```diff
@@ -19,8 +19,12 @@ func globalCmd() *cobra.Command {
 	globalCmd := &cobra.Command{}
 
 	*globalCmd = cobra.Command{
-		Use:                "global",
-		Short:              "Manage global devbox packages",
+		Use:   "global",
```

Comment:
Instead of copy-pasting, it'll be nice to use a common variable...
```
persistentPreRunE := setGlobalConfigForDelegatedCommands(globalCmd)

*globalCmd = cobra.Command{
 		Use:   "global",
 		PersistentPreRunE: persistentPreRunE,
 		 ...
 		 }
 		 
 		 ...
 addCommandAndHideConfigFlag(
 		 globalCmd,
 		servicesCmd(persistentPreRunE),
 ),
```